modules/ROOT/pages/runtime-configure-gov.adoc

```diff
@@ -0,0 +1,83 @@
+= To Configure the Mule Runtime to Connect to the Government Cloud
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+To use Mule Runtime within the Government Cloud, you must redefine the configuration properties that enable Mule Runtime to point to the EU control plane. You must also provide the Client ID and Client Secret of your environment.
+
+By default, the Mule Runtime is configured to point to the US control plane. If you do not override these properties, the Mule Runtime modules are disabled in the EU control plane.
+
+The properties you must override depend on the version of the Mule runtime you are using, as described in the following sections.
+
+== Configuring Mule Runtime 4.0
```

Comment:
I would say Mule Runtime 4.x instead, since it applies to all those versions as well.

Reply:
Or Mule Runtime 4?

Reply:
Got it - thank you!